manifests/profile/neutron/agent.pp

```diff
@@ -1,5 +1,11 @@
 # The profile to set up a neutron agent
 class openstack::profile::neutron::agent {
   include ::openstack::common::neutron
-  include ::openstack::common::ml2::ovs
+
+  if ($::openstack::config::neutron_core_plugin != "plumgrid") {
```

Comment:
The logic is a little backward here, would rather see a case statement here selecting on neutron_core_plugin so that it's easier to add more plugins later.